extensions/blocks/image-compare/view.js

```diff
@@ -54,6 +54,11 @@ domReady( function() {
 	}
 
 	function addClass( element, c ) {
+		if ( ! element ) {
+			return;
+		}
+		// depending on required browser support this
```

Comment:
Jetpack follows core's requirements and supports IE11+, so 👍

Reply:
great, refactored the old code out. Easier and cleaner to just use `classList`